pkgs/by-name/vu/vulnix/package.nix

```
@@ -8,14 +8,14 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "vulnix";
-  version = "1.12.0";
+  version = "1.12.1";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and use `build-system`